test/pylib/db/model.py

```diff
@@ -21,8 +21,8 @@ class Metric:
     memory_peak: int = None
     success: bool = None
     system_sec: float = None
-    time_end: float = None
-    time_start: float = None
+    time_end: datetime = None
+    time_start: datetime = None
```

Comment:
This doesn't seem to fit the "Integrate pytest c++..." title. Please, explain why this change is here, or keep it in separate patch.